providers/base-provider.js

```diff
@@ -88,6 +88,12 @@ function checkHash(hash, requirePrefix) {
         }
         if (bytes_1.hexDataLength(hash) === 32) {
             return hash.toLowerCase();
+        } else if (bytes_1.hexDataLength(hash) < 32) {
+            var prefix = '0x';
```

Comment:
Couldn't use `const` or `let` instead of `var`

Reply:
Using `const` or `let` throws syntax errors:

`SyntaxError: Unexpected token: name «prefix», expected: punc «;»` when using `let`

`SyntaxError: Unexpected token: keyword «const»` when using `const`